pkgs/by-name/no/nom/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "nom";
+  version = "2.0.2";
+
+  src = fetchFromGitHub {
+    owner = "guyfedwards";
+    repo = "nom";
+    rev = "v${version}";
+    hash = "sha256-6tk8NRuBbRMoaz3CmUUOC6thxIgjk/MWl50+YgQ6l5o=";
+  };
+
+  vendorHash = "sha256-fP6yxfIQoVaBC9hYcrCyo3YP3ntEVDbDTwKMO9TdyDI=";
+
+  meta = with lib; {
+    homepage = "https://github.com/guyfedwards/nom";
+    description = "RSS reader for the terminal";
+    platforms = platforms.linux ++ platforms.darwin;
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Only;
```
Just `licenses.gpl3` is deprecated